docs/Reference/API-Methods.md

```diff
@@ -2786,7 +2788,38 @@ None
         ]
     }
     ```
+### eth_submitWork
+
+Submits a Proof of Work (Ethash) solution to the blockchain.
```

Comment:
Submits a Proof of Work (Ethash) solution to the blockchain. -> Submits a Proof of Work (Ethash) solution.

I think to the blockchain is implied.

Reply:
Done.